pkg/alertmanager/api.go

```diff
@@ -89,7 +94,13 @@ func (am *MultitenantAlertmanager) SetUserConfig(w http.ResponseWriter, r *http.
 		return
 	}
 
-	cfgDesc, _ := alerts.ToProto(cfg.AlertmanagerConfig, cfg.TemplateFiles, userID)
+	cfgDesc := alerts.ToProto(cfg.AlertmanagerConfig, cfg.TemplateFiles, userID)
+	if err := validateUserConfig(cfgDesc); err != nil {
+		level.Error(logger).Log("msg", errValidatingConfig, "err", err.Error())
```

Comment:
I don't think it's an error from the cluster operator view. I would say at most a warning. Realistically, there's probably nothing the operator could do, when Cortex is running in a multi-tenant env.